### PR TITLE
Added site ID to operator key APIs

### PR DESCRIPTION
**Pull request by gmsdelmundo-ttd**

### CODE REVIEW

**Comment by gmsdelmundo-ttd** [comment]
Setting this PR as a draft until the uid2-shared version is updated.

File: src/test/java/com/uid2/admin/vertx/OperatorKeyServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            } catch (Exception ex) {
+                fail(ex);
+            }
+
+            testContext.completeNow();
+        });
+    }
+
+    @Test
+    void clientAddUnknownSiteId(Vertx vertx, VertxTestContext testContext) {
```
Comment (sunnywu) [inline src/test/java/com/uid2/admin/vertx/OperatorKeyServiceTest.java:63]: these are still tests to test adding/changing operator/operatorKey, the term "client" is a bit confusing as it seems to refer to Client Key so maybe rename it back to operatorAddUnknownSiteId instead?

Comment (gmsdelmundo-ttd) [reply src/test/java/com/uid2/admin/vertx/OperatorKeyServiceTest.java:63]: Sure, will updating this

**Review by sunnywu — APPROVED** [review]

File: src/main/java/com/uid2/admin/vertx/service/SaltService.java [hunk 2]
```diff
@@ -61,6 +61,7 @@ private void handleSaltSnapshots(RoutingContext rc) {
                     .putHeader(HttpHeaders.CONTENT_TYPE, "application/json")
                     .end(ja.encode());
         } catch (Exception e) {
+            e.printStackTrace();
```
Comment (sunnywu) [inline src/main/java/com/uid2/admin/vertx/service/SaltService.java:64]: nitpick - do you still want these printStackTrace() above/below, or do `LOGGER.error(e.getMessage(), e); `instead?

**Review by sunnywu — APPROVED** [review]

**Pull request merged**